### PR TITLE
Add more information to search results

### PR DESCRIPTION
**Pull request by yshen-google**
Fix to #1803

1. If there is no result, print "No results found".
2. Indicate that at most 10 results are returned even if there are more.

@percyliang

### CODE REVIEW

File: codalab/lib/bundle_cli.py [hunk 1]
```diff
@@ -1924,14 +1926,21 @@ def do_search_command(self, args):
             params={'worksheet': worksheet_uuid, 'keywords': args.keywords, 'include': ['owner']},
         )
 
+        print('Note: at most 10 results will be shown.')
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: Make the `10` filled in from a variable that governs the actual logic as opposed to hard-coded.

File: codalab/lib/bundle_cli.py [hunk 2]
```diff
@@ -1924,14 +1926,21 @@ def do_search_command(self, args):
...
+
         # Print direct numeric result
         if 'meta' in bundles:
+            if len(bundles['meta']['result']) == 0:
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: I don't think we need this here

File: codalab/worker/main.py [hunk 3]
```diff
@@ -89,6 +89,13 @@ def parse_args():
         'the requested image will not be downloaded. '
         'The bundle depends on this image will fail accordingly.',
     )
+    parser.add_argument(
```
Comment (percyliang) [inline codalab/worker/main.py]: This should not be part of this PR

File: codalab/lib/bundle_cli.py (hunk 1 above)
Comment (yshen-google) [reply codalab/lib/bundle_cli.py]: Good point. I moved it to a const in bundle_model.py.

File: codalab/worker/main.py (hunk 3 above)
Comment (yshen-google) [reply codalab/worker/main.py]: Yeah... it was from the previous PR. Sorry that I didn't start this branch out of clean workspace.. Since the previous PR is merged to master, this should be resolved by now.

File: codalab/lib/bundle_cli.py (hunk 2 above)
Comment (yshen-google) [reply codalab/lib/bundle_cli.py]: I see. I thought we also wanted to print that 10 results at most note when no aggregated information is returned. But I agree with you that it is needed here since it is aggregated result like count, etc. Done.

**Comment by yshen-google** [comment]
Changes:

1. Move the 10 results at most note when len(bundles) == 0.
2. Move value 10 to a const variable in bundle_model.py.

PTAL

File: codalab/lib/bundle_cli.py [hunk 4]
```diff
@@ -1931,7 +1933,10 @@ def do_search_command(self, args):
 
         # Print table
         if len(bundles) > 0:
+            print('Note: at most {} results will be shown.'.format(bundle_model.SEARCH_RESULTS_LIMIT))
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: Only print this out when `len(bundles) == SEARCH_RESULTS_LIMIT`.  Otherwise, it's the full list.

Comment (percyliang) [reply codalab/lib/bundle_cli.py]: And in that case, print "Only {} results are shown.  Use .limit=N to show N."

**Comment by nelson-liu** [comment]
Thoughts on sending the message to stderr ? I have some scripts downstream that interact with the stdout of `cl search`, and it'd be nice if existing functionality was preserved for them...

**Comment by percyliang** [comment]
Related: the extra message should only be shown when `-u` is not specified (which only prints out the raw uuids).

File: codalab/lib/bundle_cli.py (hunk 4 above)
Comment (yshen-google) [reply codalab/lib/bundle_cli.py]: Sounds good.

**Comment by yshen-google** [comment]
> Thoughts on sending the message to stderr ? I have some scripts downstream that interact with the stdout of `cl search`, and it'd be nice if existing functionality was preserved for them...

Sounds good. Moved output to stderr.

**Comment by yshen-google** [comment]
> Related: the extra message should only be shown when `-u` is not specified (which only prints out the raw uuids).

Done. This makes it ideal to have the print statement inside of print_bundle_info_list() since uuid arg is passed to that function.

**Comment by yshen-google** [comment]
Changes so far:

1. Add changes to wls in addition of search.
2. Move outputs to stderr and only print extra messages when uuid flag is set to false.

**Comment by yshen-google** [comment]
Not sure why travis-ci doesn't run at all. Filed #2191 to track.

**Comment by yshen-google** [comment]
Closed this PR due to #2191
